theme/linkform.css

```diff
@@ -26,3 +24,27 @@
 		}
 	}
 }
+
+/*
+ * Style link form differently when manual decorators are available.
+ * See: https://github.com/ckeditor/ckeditor5-link/issues/186.
+ */
+.ck.ck-link-form_layout-vertical {
+	flex-wrap: wrap;
```

Comment:
How is this supposed to work if `ckeditor5-link/theme/linkform.css` defines this class with `display: block` (not flex?) https://github.com/ckeditor/ckeditor5-theme-lark/pull/233/files#diff-e9ba6bcf18c57cfa9c06993af81ac820R54